lib/awful/widget/prompt.lua

```diff
@@ -141,7 +141,7 @@ function widgetprompt.new(args)
     promptbox.changed_callback = args.changed_callback or nil
     promptbox.keypressed_callback = args.keypressed_callback or nil
     promptbox.keyreleased_callback = args.keyreleased_callback or nil
-    promptbox.hook = args.hook or nil
+    promptbox.hooks = args.hooks or nil
```

Comment:
This spelling change should also be present on the ldoc comment [line 112](https://github.com/awesomeWM/awesome/blob/979d8eed9eac8e06d82f2d2aada3501fb38a8a28/lib/awful/widget/prompt.lua#L112) to ensure documentation consistency.